crates/tx5/src/backend.rs

```diff
@@ -0,0 +1,134 @@
+//! Backend modules usable by tx5.
+
+use std::io::Result;
+use std::sync::Arc;
+
+use futures::future::BoxFuture;
+
+use crate::{Config, PubKey};
+use tx5_core::deps::serde_json;
```

Comment:
Is rmp_serde an alternative worth considering?

Reply:
We need to have serde_json as a dependency anyways, because the webrtc signaling messages are in json. In *this* module I'm using it for the generic `Value` type. There are other generic type crates in the rust ecosystem, but if we have to have this one anyways, we should just use it.